ucsb-cs156-w24/proj-happy-cows-w24-4pm-3
# Laya render image

## Overview
In this PR, I added lower resolution background images for faster load times.

## Feedback Request (Optional)
we are supposed to come up with the optimal trade off between rendering time and aesthetic. i think the image looks the same tbh even though it's over 44 times smaller?

## Future Possibilities (Optional)
we can have a light mode/dark mode parameter?

## Validation (Optional)
1. Download this branch.
2. Run the project.
3. Go to the log in page and see if the image a) renders quickly, and b) looks nice enough

## Tests


## Linked Issues

Closes #4
